packages/story-editor/src/utils/loadStylesheet.js

```diff
@@ -19,6 +19,7 @@ function loadStylesheet(url, id) {
     const link = document.createElement('link');
     link.id = id;
     link.rel = 'stylesheet';
+    link.type = 'text/css';
```

Comment:
This should not be mandatory. It's the default.

Reply:
Fair! Added in with the config update